docker/solr/schema.xml

```diff
@@ -1123,8 +1118,16 @@
     <field name="country" type="string" indexed="true" stored="true" required="false" multiValued="false" />
     <field name="text" type="text_general" indexed="true" stored="false" required="false" multiValued="false" />
 
+    <!-- all the languages (we currently support) of OCR, mapped to the various
+         built-in language-specific field types -->
+    <field name="ocr_eng" type="text_en" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_fre" type="text_fr" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_spa" type="text_es" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_ita" type="text_it" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_ger" type="text_de" indexed="true" stored="true" required="false" multiValued="false" />
+    <copyField source="ocr_*" dest="ocr" />
```

Comment:
Yeah, I remember being mad about that when I was trying and failing to implement a true czech stemming search and not just an "english search on the czech ocr fields"